resources/js/menus/LeftMenu.vue

```diff
@@ -72,11 +72,18 @@ const { left_menu_open, clockwork_url, is_se_enabled, is_se_preview_enabled } =
 const { user } = storeToRefs(authStore);
 authStore.getUser();
 
-lycheeStore.$subscribe((_mutation, state) => {
-	if (user.value?.id) {
-		load();
-	}
-});
+watch(
+	() => user.value,
+	(newValue, oldValue) => {
+		if (newValue === null) {
+			initData.value = undefined;
+		}
+
+		if (newValue !== null && newValue.id !== oldValue?.id) {
```

Comment:
For some reason this feels neater to me. Go with what **you** prefer though as there's no real benefit to either.
```suggestion
		else if (newValue.id !== oldValue?.id) {
```